gstcefsrc.cc

```diff
@@ -326,9 +325,23 @@ class App : public CefApp
       command_line->AppendSwitch("disable-gpu-compositing");
     }
 
-    if (src->chromium_debug_port >= 0) {
-      command_line->AppendSwitchWithValue("remote-debugging-port", g_strdup_printf ("%i", src->chromium_debug_port));
+    if (src->chrome_extra_flags) {
+        gchar ** flagsList = g_strsplit((const gchar *) src->chrome_extra_flags, ",", -1);
+        for (guint i=0 ; i<g_strv_length (flagsList) ; i++) {
+
+            gchar ** switchValue = g_strsplit((const gchar *) flagsList[i], "=", -1);
+            if (g_strv_length (switchValue) > 1) {
+                GST_INFO("Adding SwitchWithValue %s=%s", switchValue[0],switchValue[1]);
```

Comment:
`GST_INFO ("Adding SwitchWithValue %s=%s", switchValue[0], switchValue[1])` please (and elsewhere too). I'm sorry for the manual indent review, but gst-indent does a very poor job with C++ I'm afraid

Reply:
by elsewhere, I mean space before function call parenthesis, and after commas